resources/artifact-ignores.properties

```diff
@@ -732,6 +732,16 @@ sinatra-chef-builder = https://www.jenkins.io/security/plugins/#suspensions
 
 # The plugin has been closed source since 2015
 starteam = https://github.com/jenkins-infra/update-center2/pull/571
+# Closed source/end of life since 2018
+openscada-jenkins-exporter = https://github.com/jenkins-infra/update-center2/pull/682
+# Closed source since 2016
+pragprog = https://github.com/jenkins-infra/update-center2/pull/682
```

Comment:
> This plugin has always been closed source since 2016

What does that mean? The latest release has a source jar: https://repo.jenkins-ci.org/artifactory/releases/org/jenkins-ci/plugins/pragprog/2.0.0/pragprog-2.0.0-sources.jar